src/ai/backend/manager/models/vfolder.py

```diff
@@ -530,13 +526,24 @@ async def prepare_vfolder_mounts(
         extra_vf_conds=extra_vf_conds,
     )
 
+    # Fast-path for empty requested mounts
+    if not accessible_vfolders:
+        return []
+    from icecream import ic
```

Comment:
is this used for debugging? `ic` is used on 559 line!

Reply:
@lizable If you want to suggest `icecream` as a tool for development setups, please add it the setup.cfg's `options.extra_require.dev` section but **_remove_** in the final PR code for review & production.

FYI: To debug-print something with its own expression, try `print(f"{expr=}")` in Python. ([ref](https://docs.python.org/3/reference/lexical_analysis.html#formatted-string-literals))
<img width="824" alt="image" src="https://user-images.githubusercontent.com/555156/167281245-84081a77-2830-432a-9b08-63766a0fb924.png">

Reply:
My mistakes. I'll fix it right away.